docs/src/main/sphinx/overview/concepts.md

```diff
@@ -81,72 +81,126 @@ for task execution.
+  Lake](/connector/delta-lake), [Hive](/connector/hive),
+  [Hudi](/connector/hudi), and [Iceberg](/connector/iceberg) connectors.
+* Connectors for relational database management systems, including the
+  [MySQL](/connector/mysql), [PostgreSQL](/connector/postgresql),
+  [Oracle](/connector/oracle), and [SQL Server](/connector/sqlserver)
+  connectors.
+* Connectors for a variety of other systems, including the
+  [Cassandra](/connector/cassandra), [ClickHouse](/connector/clickhouse),
+  [OpenSearch](/connector/opensearch), [Pinot](/connector/pinot),
+  [Prometheus](/connector/prometheus), [SingleStore](/connector/singlestore),
+  [Snowflake](/connector/snowflake) connectors.
+* A number of other utility connectors such as the [JMX](/connector/jmx),
+  [System](/connector/system), and [TPCH](/connector/tpch) connectors.
+
+Every catalog uses a specific connector. If you examine a catalog configuration
+file, you see that each contains a mandatory property `connector.name`.
 
 (trino-concept-catalog)=
 ### Catalog
 
-A Trino catalog contains schemas and references a data source via a
-connector.  For example, you can configure a JMX catalog to provide
-access to JMX information via the JMX connector. When you run SQL
-statements in Trino, you are running them against one or more catalogs.
-Other examples of catalogs include the Hive catalog to connect to a
-Hive data source.
-
-When addressing a table in Trino, the fully-qualified table name is
-always rooted in a catalog. For example, a fully-qualified table name
-of `hive.test_data.test` refers to the `test` table in the
-`test_data` schema in the `hive` catalog.
-
-Catalogs are defined in properties files stored in the Trino
-configuration directory.
+A Trino catalog is a collection of configuration properties to access a specific
+data source, including the required connector and any other details such as
+credentials and URL. Catalogs are defined in properties files stored in the
+Trino configuration directory. The name of the properties file determines the
+name of the catalog. For example, the properties file `etc/example.properties`
+results in a catalog name `example`.
+
+You can configure and use many catalogs, also using the same connector, to
+access different data sources. For example, if you have two data lakes, you can
+configure two catalogs in a single Trino cluster that both use the Hive
+connector, allowing you to query data from both clusters, even within the same
+SQL query. You can also use a Hive connector for one catalog to access a data
+lake, and use the Iceberg connector for another catalog to access the data
+lakehouse. Or you can configure different catalogs to access different
+PostgreSQL database. The combination of different catalog is determined by your
+needs to access different data sources only.
+
+A catalog contains one or more schemas, which in turn contain objects such as
+tables, views, or materialized views. When addressing an object like table in
```

Comment:
do you mean 

object-like 

Or

objects such as tables

Reply:
Updated